add-delays-to-eia-zones.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
```

Comment:
Do you want to commit this? :) In that case we can put it in `scripts`.

Reply:
Whoops no, that was not supposed to be added 🙈